torchvision/datasets/_optical_flow.py

```diff
@@ -83,7 +83,7 @@ def __rmul__(self, v: int) -> torch.utils.data.ConcatDataset:
 
 
 class Sintel(FlowDataset):
-    """`Sintel <http://sintel.is.tue.mpg.de/>`_ Dataset for optical flow.
+    """`Sintel <https://sintel.is.tue.mpg.de/>`_ Dataset for optical flow.
```

Comment:
This is one bad case. Where https leads to some other website, whereas http leads to some other.